wandb/apis/public/api.py

```diff
@@ -844,15 +844,21 @@ def reports(
     ) -> "public.Reports":
         """Get reports for a given project path.
 
-        WARNING: This api is in beta and will likely change in a future release
-
         Args:
             path: (str) path to project the report resides in, should be in the form: "entity/project"
             name: (str, optional) optional name of the report requested.
```

Comment:
this is in fact the "name" and not the display name that users would be familiar with... but should keep this for now.